.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Armeria site
+
+on:
+  push:
+    tags:
+      - armeria-*
+
+env:
+  LC_ALL: "en_US.UTF-8"
+
+jobs:
+  publish-site:
+    if: github.repository == 'line/armeria'
```

Comment:
Is it necessary? Maybe it is supposed to prevent from being published at forked repo.
However, because this workflow is triggered by tag push, it does not seem to be needed.

Reply:
Good point. But the tags could be pushed somehow to the forked repo. 
The unexpected builds could be a waste of CPU resources. Let me leave it as it is, just in case.